be/src/column/column.h

```diff
@@ -332,6 +332,8 @@ class Column {
 
     virtual Status accept_mutable(ColumnVisitorMutable* visitor) = 0;
 
+    virtual Column* mutable_child_column() { return nullptr; }
```

Comment:
Can you give this function some explanation.
But I think you can create a more general function interface.
If we support Map or Struct Column type later, this function will be a little confusing.